Edits to pgd pr5104 (#5299)
EnterpriseDB/docs: product_docs/docs/pgd/5/admin-manual/installing/05-creating-cluster.mdx

---
title: Step 5 - Creating the PGD cluster
navTitle: Creating the cluster
deepToC: true
---

## Creating the PGD cluster

* **Create connection strings for each node**.  
For each node, create a connection string that will allow PGD to perform replication.

 The connection string is a key/value string that starts with a `host=` and the IP address of the host. (If you have resolvable named hosts, the name of the host is used instead of the IP address.)

 That's followed by the name of the database. In this case, use `dbname=bdrdb`, as a `bdrdb` database was created when [installing the software](04-installing-software).

 We recommend you also add the port number of the server to your connection string as `port=5444` for EDB Postgres Advanced Server and `port=5432` for EDB Postgres Extended and community PostgreSQL.

* **Prepare the first node.** 

> To create the cluster, select and log in to one of the host Postgres server's `bdrdb` databases.

Author: Please double check this edit. It was hard to tell whether hosts was supposed to be a possessive or plural.

Reviewer: ```suggestion
To create the cluster, select and log in to the `bdrdb` database on any host's Postgres server.
```
